lab_5_scraper/scraper.py

```diff
@@ -101,6 +200,15 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    sleep(15)
```

Comment:
It is better to try different sleep values from try to try. E.g. random.randint(1,5).